src/Seo/SeoService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Sideways\Classic\Seo;
+
+use function the_seo_framework;
+
+define( 'SWCLASSIC_SITE_SCHEMA_KEY', 'sc_site_custom_schema' );
+define( 'SWCLASSIC_POST_SCHEMA_KEY', '_schema_markup' );
+
+class SeoService {
+
+    public function get_site_schema_markup(): ?string {
```

Comment:
Lets use one naming scheme, either can you change the existing code to `snake_case` or use camel-case? I don't have any preference but its good to use 1